src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/OverseasEntitySubmissionDto.java

```diff
@@ -69,6 +71,11 @@ public class OverseasEntitySubmissionDto {
     @JsonProperty("links")
     private Map<String, String> links;
 
+    @JsonIgnore
+    public boolean isForUpdate(OverseasEntitySubmissionDto overseasEntitySubmissionDto) {
```

Comment:
No need for argument.

public boolean isForUpdate() { 
    return StringUtils.isNotBlank(entity_number);
}

Reply:
If we named the method forUpdate() then would not need the @JsonIgnore. Not sure that's better.